frontend/src/Commons/ProgressBar/ProgressItem.styles.ts

```diff
@@ -0,0 +1,63 @@
+import styled from '@emotion/styled';
+import * as tokens from '@bcgov/design-tokens/js';
+import InputModuleInterface from 'src/Interface/InputModuleinterface';
+import screenSizes from '@Constants/ScreenSize';
+
+type ProgressProps = {
+  InputModule: InputModuleInterface;
+  active: boolean;
+};
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
```

Comment:
The containers have different widths, causing some shifting, `min-width: 200px` could help here

![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/80939533/f58e371e-25e8-4c2f-82a9-9e3d4e833a80)

It would also align the righ side of the progress bar with the form modules. The mockup is not aligned like this though, I suggested @Shakespeares-spy to change it.

![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/80939533/a6487715-7968-491d-b661-848ca28c2469)

Right side seems aligned by the icon, while the left seems aligned by the text

Reply:
with `min-width: 200px`:
and resizing the backbar to 85% width

![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/80939533/2f54a6f8-4008-4011-a786-adeac5f2d1f8)

Reply:
Problem with min-width is that what you see in yours is what I won't see in mine. There's different resolutions. It starts first with 1024 as our desktop resolution. But as we go bigger screens, it's not the one you're looking for.

Here's a reference on my end. It doesn't align with both left and right.
![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/144957476/f4cf4cf6-7293-4b9d-9a70-05f0a3e631f7)

Reply:
Although, this is a question that @Shakespeares-spy can answer.

Reply:
There are different sizes of widths because of the text. So there would be different gap sizes in between the flex items.

Reply:
try ` min-width: 15%;`

Reply:
So, we could use 18%, and since we only have five elments, we'd get 100% div width/5 Items.
If you'd use 20%, that would give each icon, an even 20% of the bar width. If we were to add an extra step, which our Project Owner suggested might happen already, we'd need to fit 6 icons there. That would be 16.6% width for each. So using 15% still leaves them some jiggle space and makes their min width relative to the div width, which resizes everything just nice :heart: 

Sorry for the long thaught, I just wanted to share why 15% is a good , well though of number for us to use here

Reply:
reference of min-width 15%
![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/144957476/aef27a9f-be9f-4b86-8d20-9b1ad8bbcc6e)

Reply:
As seen in the first image of this comment thread, your containers are overlapping with the progress bar beneath them. You do not want this.

Reply:
It's either we can use min-width for same sizes and ask david to change mockups. Or leave it with the first changes to align with the mockups and disregard the different gap sizes for the items.